src/test/java/teammates/test/cases/common/FieldValidatorTest.java

```diff
@@ -142,7 +142,7 @@ public void testGetValidityInfoForAllowedName() {
         
         ______TS("null value");
         
-        String typicalFieldName = "name field";
+        String typicalFieldName = "Name";
```

Comment:
With the simplified error message field name will be at the start and in bold. So no need to have the word `field` and should be capitalized.